conf/test_targeted.config

```diff
@@ -0,0 +1,21 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/sarek -profile test,<extra_test_profile>,<docker/singularity> --outdir <OUTDIR>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+try {
+    includeConfig "https://raw.githubusercontent.com/nf-core/modules/master/tests/config/test_data.config"
+} catch (Exception e) {
+    System.err.println("WARNING: Could not load nf-core/modules test data config")
+}
```

Comment:
Why do we need to have this here rather than in `nextflow.config`? We should minimise this sort of config customisation as much as possible.

Reply:
Maybe we check it once in `nextflow.config` and raise the warning from there.

Reply:
Actually left it there to check if it was needed or not, and forgot about it.

Would prefer to include that in `conf/test.config` over `nextflow.config`, it makes more sense to me to keep the test part of the config separated.